### PR TITLE
Ep/add effnetb1 cutmixpatch sam robust32 avge6e8e9e10 manylayers 324x288

### PR DESCRIPTION
**Pull request by pellegreene**

### CODE REVIEW

File: brainscore_vision/models/effnetb1_cutmixpatch_SAM_robust32_avge6e8e9e10_manylayers_324x288/requirements.txt
Comment (mike-ferguson) [inline brainscore_vision/models/effnetb1_cutmixpatch_SAM_robust32_avge6e8e9e10_manylayers_324x288/requirements.txt:1]: Also should include: 
torch
PIL
numpy

File: brainscore_vision/models/effnetb1_cutmixpatch_SAM_robust32_avge6e8e9e10_manylayers_324x288/model.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+    def forward(self, x):
+        x = self.efnet_model(x)
+        return x
+
+def get_model(name):
+    assert name == 'effnetb1_cutmixpatch_SAM_robust32_avge6e8e9e10_manylayers_324x288'
+    model_tf_efficientnet_b1_ns= EffNetBX()
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    model_tf_efficientnet_b1_ns.load_state_dict(torch.load(dir_path + "/tf_efficientnet_b1_ns_robust_cutmixpatchresize_SAM_e6e8e9e10.pth", map_location=torch.device('cpu'))["model"])
```
Comment (mike-ferguson) [inline brainscore_vision/models/effnetb1_cutmixpatch_SAM_robust32_avge6e8e9e10_manylayers_324x288/model.py:79]: weights should be uploaded to S3

**Review by mike-ferguson — CHANGES_REQUESTED** [review]
Weights should be uploaded to S3, missing some packages in requirements.txt

**Comment by pellegreene** [comment]
@mike-ferguson Should be good to merge if it all looks good.

**Pull request merged**